packages/riipen-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "riipen-ui",
-  "version": "0.0.83",
+  "version": "0.0.84",
```

Comment:
do you need me to deploy the new version when it is done?

Reply:
That would be awesome